test/runtime/Policies.go

```diff
@@ -158,6 +158,8 @@ var _ = Describe("RuntimePolicyEnforcement", func() {
 
 			By("Create a new container")
 			vm.ContainerCreate("new", helpers.HttpdImage, helpers.CiliumDockerNetwork, "-l id.new")
+			areEndpointsReady := vm.WaitEndpointsReady()
```

Comment:
Please add a statement to the commit message why this is required.

Reply:
Fixed in latest upload.